bundles/com.espressif.idf.ui/src/com/espressif/idf/ui/tools/ToolsJob.java

```diff
@@ -124,6 +124,10 @@ protected void processExportCmdOutput(final String exportCmdOp)
 		{
 			idfToolSet.setIdfVersion(matcher.group(1));
 		}
+		else 
+		{
+			idfToolSet.setIdfVersion(idfToolSet.getEnvVars().computeIfAbsent(IDFEnvironmentVariables.ESP_IDF_VERSION, k -> StringUtil.EMPTY));
+		}
```

Comment:
Not sure if this is the right way to approach the problem, I think the issue was because regular expression was not capturing the esp-idf version when it was on master branch

Reply:
Actually the output from the python is like this
```
Users/aliazamrana/.espressif/python_env/idf5.4_py3.9_env/lib/python3.9/site-packages/urllib3/__init__.py:35: NotOpenSSLWarning: urllib3 v2 only supports OpenSSL 1.1.1+, currently the 'ssl' module is compiled with 'LibreSSL 2.8.3'. See: https://github.com/urllib3/urllib3/issues/3020
  warnings.warn(
ESP-IDF v5.4-dev-3201-g46acfdce96

```
which makes it difficult to have a single regex to parse all the possibilities so it would be better I think to fallback to the env variable that is being exported by the export script

Reply:
idf.py --version output on master

```
/Users/kondalkolipaka/.espressif/python_env/idf5.4_py3.9_env/lib/python3.9/site-packages/urllib3/__init__.py:35: NotOpenSSLWarning: urllib3 v2 only supports OpenSSL 1.1.1+, currently the 'ssl' module is compiled with 'LibreSSL 2.8.3'. See: https://github.com/urllib3/urllib3/issues/3020
  warnings.warn(
ESP-IDF v5.4-dev-3201-g46acfdce96
```

on 5.2.2
```
ESP-IDF v5.2.2-dirty
```